components/02-components/header/README.md

```diff
@@ -1 +1 @@
-*There are no notes for this component yet.* 
+The header should be positioned at the top of all FEC.gov pages and connected apps.
```

Comment:
Should we add a blurb about how the homepage header looks different?

Reply:
Great idea, will do!